### DISCUSSION TITLE
Creating a Master-slave ESP-A2DP bonding

### DISCUSSION THREAD

**Discussion by christianaubry** [pschatzmann/ESP32-A2DP#222] [Q&A] [answered]
I am new to those technologies so my question may be out of scope of your project. My purpose is to create a stereo system based on 2 devices with ESP-A2DP. Hence this mean connecting the source to a first ESP32 module in master mode and having a second ESP32 already bonded to the master. Master can output the left audio channel and the slave the right channel.
 I went through your examples and documentation but could not found a way of creating the master-slave relationship.
Thanks for your help.

**Comment by pschatzmann** [comment, marked as answer]
As far as I know this is not supported by the A2DP standard und definitely not what what the API provides. You could receive both channels and stream each channel to a separate device. But streaming of audio is difficult to implement and streaming to 2 devices at the same time is even more difficult. You will need to deal with communication protocols and Codecs and the technical limitations of the ESP32! 

So nothing for the faint of heart!

**Comment by christianaubry** [comment]
Thx for your answer. I was expecting that actually as I could not get any info in the documentation.
